darts/timeseries.py

```diff
@@ -1559,8 +1594,10 @@ def quantiles_df(self, quantiles: Tuple[float] = (0.1, 0.5, 0.9)) -> pd.DataFram
         pandas.DataFrame
             The Pandas DataFrame containing the quantiles for each component.
         """
-        # TODO: there might be a slightly more efficient way to do it for several quantiles at once with xarray...
-        return pd.concat([self.quantile_df(quantile) for quantile in quantiles], axis=1)
+        if fast_mode == True:
+            return pd.concat([self.quantile_timeseries_to_df(quantile) for quantile in quantiles], axis=1)
```

Comment:
If I understand well, `quantile_timeseries_to_df()` is exactly the same method as `quantile_timeseries()`, save for the dataframe translation at the end. Couldn't we simply do:

```suggestion
            return pd.concat([self.quantile_timeseries(quantile).pd_dataframe() for quantile in quantiles], axis=1)
```

Or am I missing something?